cluster-up/cluster/kind-1.27/README.md

```diff
@@ -0,0 +1,45 @@
+# K8S 1.27.1 with mdev support in a Kind cluster
+
+Provides a pre-deployed k8s cluster with version 1.27.1 that runs using [kind](https://github.com/kubernetes-sigs/kind) The cluster is completely ephemeral and is recreated on every cluster restart.
+The KubeVirt containers are built on the local machine and are then pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+The following needs to be executed as root.
+
+```bash
+export KUBEVIRT_PROVIDER=kind-1.27
```

Comment:
You might want to remove the versioning so it won't need to be updated every now and then please
(all over the file)

Reply:
I am not quit sure about this. Do you means remove the line `export KUBEVIRT_PROVIDER=kind-1.27` in `README.md` or avoid use specific kind version in `README.md`? I think this should be fine.

Reply:
Ye just use please "x.yz", so we never need to update versioning there
Thans